script/prepare-llvm-linux.sh

```diff
@@ -22,6 +22,8 @@ cp -L llvm/bin/clang stage1/bin/
 cp -L llvm/bin/ld.lld stage1/bin/
 # a static archiver!
 cp -L llvm/bin/llvm-ar stage1/bin/
+# a LLVM configurator for PATH & LD_LIBRARY_PATH setup!
+cp -L llvm/bin/llvm-config stage1/bin/
```

Comment:
Why do we need to ship this?

Reply:
Agreed, I'll change the script to not ship `llvm-config`.